test/s3.js

```diff
@@ -39,6 +39,8 @@ describe('UploadFS S3', function () {
 
   const s3Options = {
     storage: 's3',
+    // Usually not set so we get sharp, with imagemagick fallback (the default behavior)
+    image: process.env.UPLOADFS_TEST_IMAGE,
```

Comment:
Handy if I want to run test/s3.js but force use of imagemagick. test-imagemagick.js tests the most relevant stuff though.